en/components/bulletgraph.md

```diff
@@ -110,8 +110,8 @@ Performance value is the primary measure displayed by the component and it is vi
 </igx-bullet-graph>
 ```
 
-<div class="sample-container loading" style="height: 125px">
-    <iframe id="bullet-graph-measures-iframe" data-src='{environment:demosBaseUrl}/gauges/bullet-graph-measures' width="100%" height="100%" seamless="" frameBorder="0" class="lazyload"></iframe>
+<div class="sample-container" style="height: 125px">
+    <iframe id="bullet-graph-measures-iframe" src='{environment:demosBaseUrl}/gauges/bullet-graph-measures' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
```

Comment:
who is making these changes and why?

Reply:
As I see these changes are comming from `tfsbuild` [commit 120bbdac42e5f663ea725fffa1d11a12d25e8eb4](https://github.com/IgniteUI/igniteui-docfx/pull/1156/commits/120bbdac42e5f663ea725fffa1d11a12d25e8eb4).

As I've said, they are not wrong, its just they remove the lazy loading functionality of the iframe. If you want we can leave these changes, it is fine by me.